lib/adapters/user_adapter.ts

```diff
@@ -14,5 +14,5 @@ export const userAdapter = (cu: Collibra.User): E.Either<string[], User> =>
       lastName: cu.lastName,
       updatedAt: new Date(cu.lastModifiedOn),
     } as User,
```

Comment:
Refactor in the same way as role_adapter with the object creating outside of the pipe

Reply:
fair